tensorflow_lite_support/ios/BUILD

```diff
@@ -66,17 +71,61 @@ tflite_ios_framework(
     hdrs = [
         ":base_options.h",
         ":bounding_box.h",
+        ":category.h",
         ":classification_options.h",
         ":classification_result.h",
+        ":detection_result.h",
+        ":segmentation_result.h",
         ":common.h",
         ":frame_buffer.h",
         ":image_classifier.h",
+        ":object_detector.h",
+        ":image_segmenter.h",
     ],
     allowlist_symbols_file = ":allowlist_TensorFlowLiteTaskVision.txt",
     bundle_name = "TensorFlowLiteTaskVisionC",
     minimum_os_version = TFL_MINIMUM_OS_VERSION,
     deps = [
         "//tensorflow_lite_support/c/task/vision:image_classifier",
+        "//tensorflow_lite_support/c/task/vision:object_detector",
+        "//tensorflow_lite_support/c/task/vision:image_segmenter",
+    ],
+)
+
+tflite_ios_framework(
```

Comment:
@priankakariatyml You don't need one tflite_ios_framework for each task. Just having a single `TensorFlowLiteTaskVisionC_framework` is enough.